src/test/java/uk/gov/register/functional/FindEntityTest.java

```diff
@@ -18,17 +18,17 @@
 public class FindEntityTest {
 
     @ClassRule
-    public static RegisterRule register = new RegisterRule("address");
+    public static RegisterRule register = new RegisterRule();
 
     @Before
     public void publishTestMessages() {
         register.wipe();
-        register.mintLines("{\"street\":\"ellis\",\"address\":\"12345\"}", "{\"street\":\"presley\",\"address\":\"6789\"}", "{\"street\":\"ellis\",\"address\":\"145678\"}");
+        register.mintLines("address", "{\"street\":\"ellis\",\"address\":\"12345\"}", "{\"street\":\"presley\",\"address\":\"6789\"}", "{\"street\":\"ellis\",\"address\":\"145678\"}");
     }
 
     @Test
     public void find_shouldReturnEntryWithThPrimaryKey_whenSearchForPrimaryKey() throws JSONException {
-        WebTarget target = register.target();
+        WebTarget target = register.targetRegister("address");
```

Comment:
missing 'e' in the test name.... I know it's not you but I'll let you have a joy of finding it :)